examples/lighting-app/lighting-common/lighting-app.zap

```diff
@@ -2981,6 +2981,22 @@
           "side": "client",
           "enabled": 0,
           "commands": [
+            {
+              "name": "AttestationRequest",
+              "code": 0,
+              "mfgCode": null,
+              "source": "client",
+              "incoming": 1,
+              "outgoing": 1
+            },
+            {
+              "name": "CertChainRequest",
```

Comment:
nit: This should be `CertificateChainRequest` and `CertificateChainResponse` according to spec.

Reply:
will update